zend_scoutapm.c

```diff
@@ -185,7 +185,9 @@ static PHP_RINIT_FUNCTION(scoutapm)
     SCOUTAPM_G(disconnected_call_argument_store) = calloc(0, sizeof(scoutapm_disconnected_call_argument_store));
     SCOUTAPM_DEBUG_MESSAGE("Stacks made\n");
 
+#ifndef ZTS
```

Comment:
Hmm, something tells me this isn't the right fix :/ basically I don't want to override the handlers more than once, but I think in ZTS mode for some reason the `handlers_set` is already set so doesn't override the handlers, hence why all the tests are failing in #67 - not sure why we've not seen this behaviour before though...

Reply:
This change shows that `handlers_set` is not properly **initialized**, which should be fixed by PR #69